### PR TITLE
Compare core

### PR DESCRIPTION
**Pull request by vogella**

### CODE REVIEW

File: bundles/org.eclipse.compare.core/src/org/eclipse/compare/internal/core/patch/PatchReader.java [hunk 1]
```diff
@@ -165,14 +163,12 @@ private String readUnifiedDiff(List<FilePatch2> diffs, LineReader lr, String lin
...
 		List<FilePatch2> diffs= new ArrayList<>();
-		boolean reread= false;
+		boolean reread= line!=null;
 		String diffArgs= null;
 		String fileName= null;
 		List<String> headerLines = new ArrayList<>();
 		boolean foundDiff= false;
 
-		// read leading garbage
-		reread= line!=null;
```
Comment (vogella) [inline bundles/org.eclipse.compare.core/src/org/eclipse/compare/internal/core/patch/PatchReader.java:175]: comment lost, will update

**Review by vogella — COMMENTED** [review]
comment lost

**Comment by vogella** [comment]
@akurtakov any reasons why this should be still on Java 8?

**Comment by akurtakov** [comment]
Not that I know of.

**Pull request merged**